extension/src/popup/components/WarningMessages/styles.scss

```diff
@@ -154,14 +154,8 @@
 
     // TODO - use new SDS button when ready
     .BasicButton--primary {
-      background: var(--pal-error-background);
-      border-color: var(--pal-error-border);
-      --Button-color-background: var(--pal-error-background);
-      --Button-color-border: var(--pal-error-border);
-    }
-    .BasicButton--primary:hover {
-      background-color: var(--pal-error);
-      border-color: var(--pal-error);
+      background: #e5484d;
+      border-color: #e5484d;
```

Comment:
The new "red" button uses a different color according to the figma

Reply:
ah nice catch, we'll still need the hover state though. Otherwise the color goes blue on hover

<img width="360" alt="Screen Shot 2022-10-24 at 11 55 40 AM" src="https://user-images.githubusercontent.com/30449853/197571842-60985a23-7af1-4dd7-95f3-defa10f2ec1d.png">

not sure what the hover background color should be, maybe this?
```
    .BasicButton--primary:hover {
      background: #e5484d;
      border-color: #e5484d;
      background-color: var(--pal-error);
    }
```